xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/register_macros.vm

```diff
@@ -0,0 +1,219 @@
+## Defines what server generated error messages should look like
+## The error message when a field is entered incorrectly
+#set ($failureMessageParams = {'class': 'LV_validation_message LV_invalid'})
+## 'LV_validation_message LV_invalid' depends on this:
+$xwiki.get('ssfx').use('uicomponents/widgets/validation/livevalidation.css', true)
+##
+## The * next to the fields to denote they are mandatory.
+#set ($fieldMandatoryStar = {'class': 'xRequired'})
+##
+#macro (definePasswordFields, $fields, $passwordFieldName, $confirmPasswordFieldName)
+  #set ($field =
+    {'name': $passwordFieldName,
+      'label': $services.localization.render('core.register.password'),
+      'params': {
+        'type': 'password',
+        'autocomplete': 'off',
+        'size': '60'
+      },
+      'validate': {
+        'mandatory': {
+          'failureMessage': $services.localization.render('core.validation.required.message')
+        },
+        'regex': {
+          'pattern': '/.{6,}/',
+          'failureMessage': $services.localization.render('xe.admin.registration.passwordTooShort')
+        }
+      }
+    })
+  #set ($discard = $fields.add($field))
+##
+## The confirm password field, mandatory, must match password field, and must also be 6+ characters long.
```

Comment:
Missing indentation?

Reply:
From what I understood from @mflorea for templates, the blank spaces at the beginning of lines will not be removed. Considering that, spaces before the comments will result in a blank space.